sample-apps/python-auto-instrumentation-sample-app/app.py

```diff
@@ -34,23 +35,25 @@
 # variable to keep track of the total number api requests
 n = 0
 
+common_attributes = { 'signal': 'metric', 'language': 'python-auto-instrumentation', 'metricType': 'request' }
```

Comment:
Dumb question but what are these common_attributes used for, besides (what I assume to be) being required for the spec? I also notice that this is for metrics only, and for traces they are individually hardcoded for each span, is there a particular reason why this is?

Reply:
These common_attributes are tested for in the testcases for metrics as they are added to the metric instruments in the sample app when they are updated.